docs/i18n.md

```diff
@@ -0,0 +1,7 @@
+# i18n
+
+We use [react-i18next](https://react.i18next.com/) to handle internationalization.
+
+Locales are stored in `packages/web/src/public/locales/{language-code}/{namespace}.json`
+
+For now, we have only one namespace `translation`. This will expand as needed.
```

Comment:
This is very slim for now, but eventually we will want to document our expectations for how developers should think about internationalization. That's going to take time to develop as we develop experience through the MVP